website_authorized_service_centers/views/layout.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
+  <template id="assets_frontend" inherit_id="website.assets_frontend" name="Yoytec Service Providers Frontend Assets">
```

Comment:
Here you are injecting the css to all the site, when you need it only for 1 page, can you fix that please.